controllers/managedmcg_controller_test.go

```diff
@@ -181,8 +205,6 @@ func TestManagedMCGReconcilerReconcile(t *testing.T) {
 	if err != nil {
 		t.Errorf("ManagedMCGReconciler.Reconcile() error: %v", err)
 	}
-	if err := r.removeNoobaa(); err != nil {
-		t.Errorf("Error while removing Nooba: %v", err)
-	}
+	miscTest(t, r)
```

Comment:
Do we _need_ to introduce a new function here? I'd suggest unwrapping the function contents here.

Reply:
Yeh, done that